code/RedirectedURLCsvBulkLoader.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Created by priyashantha@silverstripers.com
+ * Date: 4/19/18
+ * Time: 3:23 PM
+ */
+
+class RedirectedURLCsvBulkLoader extends CsvBulkLoader
```

Comment:
This bulk loader needs to be registered against the appropriate assetadmin

Reply:
You meant modeladmin, yeah. I've forgot to add it in model importers as I was editing origin `CSVBulkLoader.php`. Just fixed it.